### ISSUE TITLE
Improve Accuracy of Standard Classification Approach

### ISSUE THREAD

**Issue by WGierke** [WGierke/git_better#9]
![download 5](https://cloud.githubusercontent.com/assets/6676439/20865426/0c59153c-ba1a-11e6-9464-2cca1724cdf9.png)

When fitted on the training data, classifiers such as XGBoost or RandomForest yield an accuracy of over 80% while they only yield one of 20% when they classify the validation data ([see here](https://github.com/WGierke/git_better/blob/master/Repository_Classification.ipynb)).
Obviously that's overfitting but it would be great if you could investigate how we can improve their accuracy (NLP on description yields 51% at the moment).
